### PR TITLE
[MIG] sale_partner_delivery_window: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+odoo14-addon-partner-tz @ git+https://github.com/trobz/partner-contact@14.0-mig-partner_tz#subdirectory=setup/partner_tz
+odoo14-addon-base-time-window @ git+https://github.com/trobz/server-tools@14.0-mig-base_time_window#subdirectory=setup/base_time_window
+odoo14-addon-stock-partner-delivery-window @ git+https://github.com/trobz/stock-logistics-workflow@14.0-mig-stock_partner_delivery_window#subdirectory=setup/stock_partner_delivery_window
```
Comment (sebalix) [inline test-requirements.txt]: Dependencies have been merged:
- https://github.com/OCA/edi/pull/332
- https://github.com/OCA/server-tools/pull/1988
- https://github.com/OCA/stock-logistics-workflow/pull/775

**Review by sebalix — COMMENTED** [review]
Runbot was complaining, I restarted it to check.
Dependencies to clean as well, but LG

File: test-requirements.txt (hunk 1 above)
Comment (dzungtran89) [reply test-requirements.txt]: thx, I updated it

Comment (sebalix) [reply test-requirements.txt]: My bad, we were missing `partner_identification_import` when I commented, but now it is merged, you can remove this file now :)

Comment (dzungtran89) [reply test-requirements.txt]: Hi @sebalix , I misunderstood too :(. I just pushed the update

**Comment by dzungtran89** [comment]
something wrong with the travis build, we're checking ...

**Comment by hailangvn** [comment]
Good day, the travis and runbot are both green now.

**Comment by mmequignon** [comment]
superseded by https://github.com/OCA/sale-workflow/pull/1736

**Pull request closed without merging**